inspire_dojson/hep/rules/bd9xx.py

```diff
@@ -305,6 +305,11 @@ def references2marc(self, key, value):
     a_values.extend(['hdl:' + el['value'] for el in pids if el.get('schema') == 'HDL'])
     a_values.extend(['urn:' + el['value'] for el in pids if el.get('schema') == 'URN'])
 
+    eids = force_list(reference.get('external_system_identifiers'))
+    u_values = force_list(get_value(reference, 'urls.value'))
+    u_values.extend(['https://cds.cern.ch/record/' + el['value'] for el in eids if el.get('schema') == 'CDS'])
```

Comment:
* you should use `format` instead of string concatenation, and put the URL in a constant.
So `CDS_RECORD_FORMAT = 'https://cds.cern.ch/record/{}'` at the top, and here
`CDS_RECORD_FORMAT.format(el['value'])`.
* `extend` can take an iterable, so you don't have to put the square brackets.